### PR TITLE
Added footer "about" text

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* None

*Description of changes:*
- updated version to 0.3.4
- added about text to footer

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/package-lock.json [hunk 1]
```diff
@@ -2666,98 +2666,6 @@
         "tslint": "^5.20.1",
         "webpack": "^4.0.0",
         "yorkie": "^2.0.0"
-      },
-      "dependencies": {
```
Comment (severinbeauvais) [inline client/package-lock.json:2670]: This is result of a new `npm i`. Looks like npm just cleaned things up since the last time.

**Review by vysakh-menon-aot — APPROVED** [review]

File: client/package.json [hunk 2]
```diff
@@ -1,7 +1,9 @@
 {
   "name": "name-request",
-  "version": "0.3.3",
+  "version": "0.3.4",
   "private": true,
+  "appName": "Name Request UI",
+  "sbcName": "SBC Common Components",
```
Comment (cameron-freshworks) [inline client/package.json:6]: Cool, haven't seen this one before.

**Review by cameron-freshworks — APPROVED** [review]
Simple and neat!

File: client/package.json (hunk 2 above)
Comment (severinbeauvais) [reply client/package.json:6]: I added this to our other projects a while back.

**Pull request merged**